03/GeoffRiley/tags.py

```diff
@@ -0,0 +1,41 @@
+import re
+from collections import Counter
+from difflib import SequenceMatcher
+from itertools import permutations
+
+TOP_NUMBER = 10
+RSS_FEED = 'rss.xml'
+SIMILAR = 0.87
+
+TAG_REGEX = re.compile(r'<category>([^<]+?)</category>')
+
+
+def get_tags() -> list:
+    """Find all tags in RSS_FEED.
+    Replace dash with whitespace."""
+    with open(RSS_FEED) as f:
+        return [t.replace('-', ' ').lower() for t in TAG_REGEX.findall(f.read())]
+
+
+def get_top_tags(tags):
+    """Get the TOP_NUMBER of most common tags"""
+    return list(Counter(tags).most_common(TOP_NUMBER))
+
+
+def get_similarities(tags):
```

Comment:
as you return a list of tag pairs, the function name is quite misleading in my opinion, because similarity sounds like a metric and I would expect to get the similarity metric of passed tags and not a list of tag pairs of two.

Reply:
This is another case of following the template :smile: